target: [0.5, 0.5, 0.5]
orientation: [0.0, 0.0, 0.0]
sub_topic: /gen3/joint_states
urdf_filename: urdf/gen3.urdf